pkg/security/podsecuritypolicy/provider.go

```diff
@@ -94,13 +94,13 @@ func (s *simpleProvider) MutatePod(pod *api.Pod) error {
 
 	// This is only generated on the pod level.  Containers inherit the pod's profile.  If the
 	// container has a specific profile set then it will be caught in the validation step.
-	seccompProfile, err := s.strategies.SeccompStrategy.Generate(pod.Annotations, pod)
+	seccompProfile, err := s.strategies.SeccompStrategy.Generate(pod.GetAnnotations(), pod)
 	if err != nil {
```

Comment:
I am not familiar with this codebase. But I am confused that why should  we use Get and Set method instead of reading/setting pod directly?

Reply:
Now that Get and Set functions have already been provided, we could use it.